test/FunctionalTests/HealthChecks.UI/Configuration/UIConfigurationTests.cs

```diff
@@ -319,5 +321,39 @@ public void allow_disable_running_database_migrations_using_configuration_provid
 
             UISettings.DisableMigrations.Should().Be(true);
         }
+
+        [Fact]
+        public async Task support_configuring_page_title()
+        {
+            var pageTitle = "My Health Checks UI";
+
+            var builder = new WebHostBuilder()
+                .UseStartup<DefaultStartup>()
```

Comment:
😕  A "small" problem here. I removed `DefaultStartup` class some time ago so this code does not compile...
Actually `FunctionalTests.csproj` was lost from sight. Maybe `HealthChecks.UI.Tests` is a right place to go (most likely). I don't know exactly but it seems that project was renamed/copied some time ago. Also tests should be enabled for UI workflow here https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/master/.github/workflows/healthchecks_ui_ci.yml#L148 . Now the tests do not start at all.

ping @carlosrecuero 

@gabrielmaldi Small changes lead to great work 😉 .

Reply:
Done.

Reply:
I hadn't noticed all this, I just rebased and pushed, sorry!

Now I moved my new test to the right project, deleted the obsolete FunctionalTests folder (in a separate commit if you wish to revert it), and enabled the UI workflow tests in the file you pointed to.

Thanks!

Reply:
UI tests hang. I have not yet found an answer.